blog/2025-01-17-january.md

```diff
@@ -7,6 +7,8 @@ tags: [media, Developer Office Hours]
 
 This 12th installment of the Cardano Developer Office Hours is presented by Satya Ranjan, Senior Enterprise Architect at the Cardano Foundation. In this session, Satya discusses how the yaci-devkit simplifies building and testing on a local Cardano dev network, helping developers iterate faster and avoid public testnet limitations. The session highlights key features such as customizable block times, integrated tools, and Blockfrost-compatible APIs. A live demo shows easy setup using Docker and NPM distributions.
 
+<!-- truncate -->
```

Comment:
The posts tagged with "media" are very short and not ment to be truncated otherwise you can't see the video when scrolling through https://developers.cardano.org/blog/ which gives an odd experience.